src/block/progress-bar/edit.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Internal dependencies
+ */
+import ProgressBarStyles from './style'
+import { DEFAULT_PERCENT } from '../../block-components/progress-bar/attributes'
```

Comment:
I just realized that this is a reference to `src/block-components` which is built to a separate module from this file `stk.js`, this might include duplicate code into our `editor_blocks.js`.

Please add another declaration for this value somewhere in `src/block/progress-bar` instead, so we won't have to reference `src/block-components`